packages/app/app/stacks/page.tsx

```diff
@@ -11,7 +11,7 @@ export default function Page() {
   if (isDisconnected) return <NoWalletState />;
 
   return (
-    <div className="space-y-8">
+    <div className="py-10 space-y-8">
```

Comment:
I don't think this spacing is good because we already doing padding top spacing on layout, we should do the spacing y there.

Reply:
We should converge the spacing with the home page.

It should have the same spacing top.

We probably should do this spacing on the `RootLayout` which is shared by the two pages.

```jsx
<div className="px-4 py-12 md:py-16 mx-auto md:px-0">{children}</div>
```

And remove spacing top from `YourStacksLayout`

Reply:
+1 to normalizing spacing, will do!